Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
```

Comment:
Instead of having hard coded different Dockerfiles for bionic and focal, you could do something like

```
ARG ubuntu_version
FROM ubuntu:${ubuntu_version}

# ...
```
The images for 18.04 and 20.04 can then be built with 
```
docker build --build-arg ubuntu_version=18.04 ...
```
and 
```
docker build --build-arg ubuntu_version=20.04 ...
```

This has the upside that we don't replicate the Dockerfile and only have to adjust the CI pipeline to build upgrade images for different ubuntu versions.

Reply:
Didn't know that worked! Neat, implemented that.